ModelCatalogueCorePlugin/grails-app/assets/javascripts/modelcatalogue/core/orderedMapEnhancer.coffee

```diff
@@ -15,8 +15,7 @@ angular.module('mc.core.orderedMapEnhancer', ['mc.util.enhance']).config ['enhan
             if value.key == key
               value.value = newValue
               return newValue
-          # only push if the value is defined
-          self.values.push(key: key, value: newValue) if newValue or not key
+          self.values.push(key: key, value: newValue)
```

Comment:
@musketyr do you know why this was added here? I've searched in git history but wasn't able to find an issue number. For me, it make sense to allow adding null values as well. At least from import perspective :-) :8ball: